src/App.tsx

```diff
@@ -2,12 +2,14 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { appName } from './common/config';
 import MainPage from './pages/MainPage';
+import Home from './pages/Home';
 
 import './styles/App.scss';
 
 function App() {
   return (
     <>
+      <Home/>
```

Comment:
It would be better to use only this component and remove `MainPage`, for correct rendering